### DISCUSSION TITLE
How to Build a CRUD Application in Minutes Budibase YouTube July 14th, 2021 User Jams

### DISCUSSION THREAD

**Discussion by LugarJE** [Budibase/budibase#2867] [Help] [answered]
Newbies follow BB How to link repeatedly, here at https://www.youtube.com/watch?v=DsBudu6Kzls, just to ensure you can follow the lesson (many 5 second rewinds are in order). Then if repeatable use the @time to tell how far you got and what your issues you are having. Also, include how you are running it local self host, Docker, OS (Windows, mac, Linux). Then may help come from this fine community.

Shout out to Joe Budi (Is that your real name?-- be so cool if it is!)

**Comment by LugarJE** [comment]
Issue: No Columns to be bound to Books search @12:33
Allow me to be first, I'm self-hosting BB locally with Docker, on Debian Buster (HP Laptop). When I pick on lightening bolt, I only get 3 things: 1) Search window 2) Devices.Mobile 3) Devices.Tablet 4) Text box (to type a value?) after stopping video just before save, tried typing in [[New Form.Fields.Title]] but get no rows found after the save (@12:40) I'm using the August Release BB.

**Reply by LugarJE** [reply]
Partial success, I typed in {{New Form.Fields.Title}} with curly braces, and this allowed the search to work, but I still didn't get the same Columns option Buddy Joe Budi shows in video.

**Comment by mjashanks** [comment, marked as answer]
Hi John, welcome.

Joe's name is not "Joe Budi", but I might call him that from now on! 

If you don't have `{{ New Form.Fields.Title }}` as a selectable field, that would suggest that your data provider does not have access to the `Form`. Ensure that your `Data Provider` is a child/grandchild of your form. For the filter to work, that field needs to be selectable - free typing won't do it.

**Reply by LugarJE** [reply]
Probably such a newbie! I have a small screen. I did check the video again and your recommendation; they were as described! So, I was able to scroll lower in below my devices Mobile/tablet (#2 and #3 above), and found the rest of my objects selectable from Design folders at left of app from /books and below. Perhaps a bit in tunnel vision on my laptop screen. HTH some other Laptop user (or newb). Thanks for the reply.
